src/averageAges.js

```diff
@@ -20,6 +20,19 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const callbackFilterMenIncludesCentury = (person) => person.sex === 'm'
+  && Math.ceil(person.died / 100) === century;
+  const callbackFilterMen = (person) => person.sex === 'm';
+  const callbackAge = (person) => person.died - person.born;
+  const callbackSum = (person1, person2) => person1 + person2;
+
+  const mens = century !== undefined
+    ? people.filter(callbackFilterMenIncludesCentury).map(callbackAge)
+    : people.filter(callbackFilterMen).map(callbackAge);
```

Comment:
better to use ternary operator inside callback function.
You can use destructuring to avoid repeating person.
For example:

```
const men = people.filter(({ sex, died, born }) => century
      ? sex === 'm' && Math.ceil(died / 100) === century
      : sex === 'm';
```

Reply:
Tre to replace using filter and map to just one method reduce

Reply:
> 

Could you show example?